### PR TITLE
Improve helix tutorial and code formatting (#1931)

### PR DESCRIPTION
**Pull request by qqu0127**
### Issues

- [x] My PR addresses the following Helix issues and references them in the PR description:
Fixed issue #1931 , improve Helix tutorial and minor changes to code formatting and comments. 

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
Improve Helix tutorial and minor changes to code formatting and comments. 

### Tests

- [x] The following tests are written for this issue:
mvn test

- The following is the result of the "mvn test" command on the appropriate module:
All test passed

### Commits

- My commits all reference appropriate Apache Helix GitHub issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Code Quality

- My diff has been formatted using helix-style.xml 
(helix-style-intellij.xml if IntelliJ IDE is used)

### CODE REVIEW

File: website/0.9.8/src/site/markdown/tutorial_task_framework.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ under the License.
 
 ## [Helix Tutorial](./Tutorial.html): Task Framework
 
-Task framework, in Helix, provides executable task scheduling and workflow management. In Helix, three layers of task abstraction have been offered to user for defining their logics of dependencies. The graph shows the relationships between three layers. Workflow can contain multiple jobs. One job can depend on other one. Multiple tasks, including same task different partition and different task different partition, can be added in one job.
+Task framework, in Helix, provides an executable task scheduling and workflow management. In Helix, three layers of task abstraction have been offered to user for defining their logics of dependencies. The graph shows the relationships between three layers. Workflow can contain multiple jobs. One job can depend on other one. Multiple tasks, including same task different partition and different task different partition, can be added in one job.
```
Comment (narendly) [inline website/0.9.8/src/site/markdown/tutorial_task_framework.md:26]: No particle "an" needed :)

File: website/0.9.8/src/site/markdown/tutorial_task_framework.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ under the License.
...
 Task framework not only can abstract three layers task logics but also helps doing task assignment and rebalancing. User can create a workflow (or a job queue) at first beginning. Then jobs can be added into workflow. Those jobs contain the executable tasks implemented by user. Once workflow is completed, Helix will schedule the works based on the condition user provided.
 
 ![Task Framework flow chart](./images/TaskFrameworkLayers.png)
 
 ### Key Concepts
-* Task is the basic unit in Helix task framework. It can represents the a single runnable logics that user prefer to execute for each partition (distributed units).
+* Task is the basic unit in Helix task framework. It represents a single runnable logics that user prefer to execute for each partition (distributed units).
```
Comment (narendly) [inline website/0.9.8/src/site/markdown/tutorial_task_framework.md:32]: "Task is the smallest unit of work in Helix Task Framework..." might be more intuitive.

File: website/0.9.8/src/site/markdown/tutorial_task_framework.md (hunk 1 above)
Comment (qqu0127) [reply website/0.9.8/src/site/markdown/tutorial_task_framework.md]: thanks, fixed in new commit.

File: website/0.9.8/src/site/markdown/tutorial_task_framework.md (hunk 2 above)
Comment (qqu0127) [reply website/0.9.8/src/site/markdown/tutorial_task_framework.md]: thanks, fixed in new commit.

**Pull request merged**